inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/InlongStream.java

```diff
@@ -51,6 +51,16 @@ public interface InlongStream {
      */
     Map<String, StreamSink> getSinks();
 
+    /**
+     * Get detail info of data sink by id.
+     */
+    StreamSink getSinkInfoById(Integer sinkId);
+
+    /**
+     * Get detail info of data sink by name.
+     */
+    StreamSink getSinkInfoByName(String sinkName);
```

Comment:
The sink name was not unique in the global scope, is this method work fine?

Reply:
This is obtained from the SinkMap in StreamImpl, and it is not queried from the database. The duplicate name check has been performed when the map is generated.